packages/ui-test-sandbox/src/utils/sandbox.ts

```diff
@@ -150,7 +146,6 @@ class Sandbox {
       })
     }
 
-    this._observer.observe(document.head, { childList: true })
```

Comment:
This is the real change in this PR.
Observing the the `<head>` element changes was not working well with Emotion:
For performance reasons emotion tries to insert the generated css into the `head` element of the document.
If we do this `observe` here, then after every test case the `teardown` method would remove all of the nodes which were injected by emotion.
This caused issues with emotion being able to insert elements to the `head` thus deleting the observation was the only change that allowed us to do the `(node as ChildNode).remove()` below which then only deletes the nodes inside the `body` of the document.